plugins/yellow-core/skills/create-agent-skills/references/subagent-failure-convention.md

```
@@ -0,0 +1,168 @@
+## Subagent Failure Convention (Output-File Pattern)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)